composer.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name"			: "ipub/flash-messages",
+	"name"			: "vymak/flash-messages",
```

Comment:
i do not think so :)

Reply:
Sorry I changed it on bad branch, I will change it on another branch :)